locales/ru-RU.json

```diff
@@ -415,22 +415,22 @@
     },
     "enter_roundabout": {
       "phrases": {
-        "0": "Выезжайте на кольцевую развязку.<instructions.enter_roundabout.phrases.1>Выезжайте на кольцевую развязку и сверните на <ORDINAL_VALUE> съезде.",
```

Comment:
its interesting that this doesnt throw an error in the unit tests. we should make a new issue to harden the tests to this. no only should we find the tags in the phrase that we expect we should also not find additional tags in the phrase that shouldnt be there. in this case someone copy past failed two phrases together... but a stronger unit test would catch this.